kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ActiveAddFilesIterator.java

```diff
@@ -140,12 +153,15 @@ private void prepareNext() {
         if (!isFromCheckpoint) {
             final ColumnVector removesVector = addRemoveColumnarBatch.getColumnVector(1);
             for (int rowId = 0; rowId < removesVector.getSize(); rowId++) {
-                if (removesVector.isNullAt(rowId)) continue;
+                if (removesVector.isNullAt(rowId)) {
+                    continue;
+                }
 
                 // Note: this row doesn't represent the complete RemoveFile schema. It only contains
                 //       the fields we need for this replay.
 
                 final Row row = removesVector.getStruct(rowId);
+                // Make this a method on the `RemoveFile`.
```

Comment:
is this supposed to be a `TODO: make this a method on the RemoveFile`?

Reply:
no longer needed.